Dockerfile

```diff
@@ -1,4 +1,20 @@
-# This image is based on the official sonar-scanner image
+# Builder image for other analysis tools
+FROM sonarsource/sonar-scanner-cli:4.4 AS builder
+
+# CppCheck
+ADD https://netix.dl.sourceforge.net/project/cppcheck/cppcheck/1.90/cppcheck-1.90.tar.gz \
+    .
+
+RUN apk add --no-cache \
```

Comment:
Codacy found an issue: [Use WORKDIR to switch to a directory](https://app.codacy.com/gh/cnescatlab/sonar-scanner/pullRequest?prid=6081579)